ogr/services/github/pull_request.py

```diff
@@ -123,14 +123,19 @@ def create(
         source_branch: str,
         fork_username: str = None,
     ) -> "PullRequest":
+        """        
+        The default behaviour is pull request is made to the immediate parent repository if the repository
+        is a forked repository.
+        If you wanto create an pull request to the forked repo, please pass the `fork_username` parameter. 
```

Comment:
Flake does not like long lines, can you please split the text?

https://softwarefactory-project.io/zuul/t/local/build/159c54dd94e5407c9eb40976728d0769/console:

```
flake8...................................................................Failed
- hook id: flake8
- exit code: 1

ogr/services/github/pull_request.py:127:101: E501 line too long (106 > 100 characters)
ogr/services/github/pull_request.py:129:101: E501 line too long (106 > 100 characters)
```